tools/pipelines/server-gitrest.yml

```diff
@@ -45,12 +35,9 @@ trigger:
     - tools/pipelines/templates/build-docker-service.yml
     - tools/pipelines/templates/include-set-package-version.yml
     - tools/pipelines/templates/include-vars.yml
-    - tools/pipelines/templates/include-generate-notice-steps.yml
```

Comment:
looks like you have some bad merges judging from the changes here

Reply:
@chensixx this PR seems to be reverting some changes I made to server/gitrest earlier this week. Was that intentional? We need those changes to be there.

Reply:
hmm weird, this was not intentional